arch/arm/src/stm32h7/stm32_gpio.c

```diff
@@ -162,6 +162,20 @@ int stm32_configgpio(uint32_t cfgset)
       return -EINVAL;
     }
 
+#if (STM32H7_NGPIO > 5) && (!defined(CONFIG_STM32H7_HAVE_GPIOF))
```

Comment:
Can all this be just a null check on line 181?

Reply:
I can init g_gpiobase[5] and g_gpiobase[6] to 0 and check for that every place g_gpiobase[] is used. Will that be acceptable?

Reply:
I made this change in commit 7d70abc4d4a1. Because g_gpiobase[5] and g_gpiobase[6] is initted to 0, a null check is now required in stm32_gpioread() and stm32_gpiowrite(). I added those and fixed resulting nxstyle complaints.

Waiting to hear your thoughts... Thanks